app/views/api/v2/hosts/main.json.rabl

```diff
@@ -10,7 +10,7 @@ extends "api/v2/hosts/base"
 attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_id, :realm_name,
            :sp_mac, :sp_ip, :sp_name, :domain_id, :domain_name, :architecture_id, :architecture_name, :operatingsystem_id, :operatingsystem_name,
            :subnet_id, :subnet_name, :sp_subnet_id, :ptable_id, :ptable_name, :medium_id, :medium_name, :build,
-           :comment, :disk, :installed_at, :model_id, :hostgroup_id, :hostgroup_name, :owner_id, :owner_type,
```

Comment:
@ohadlevy I changed `hostgroup_name` to show `hostgroup.name`.
Right now `hostgroup_name` actually displays `hostgroup.title` so my change can be considered breaking the api, but right now the attribute is not showing what it claims to be showing.
I can just add `hostgroup_title` and then we will have 2 identical attributes with different names.

 +1 for changing it completely (in v3 at least), especially since when I try to look at the output I'd prefer seeing all the `hostgroup_...` together.